src/@carbon-info/views/Data/Data.tsx

```diff
@@ -1,217 +1,243 @@
 import { FadeAndSlide } from "@carbon-info/components";
 import { useContentful } from "@carbon-info/hooks";
 import { isWidth } from "@carbon-info/utils/environment";
-import { StyleUtils } from "@carbon-info/utils/styles";
-import { Divider, Grid, makeStyles, Paper, Theme, Typography } from "@material-ui/core";
+import {
+	Box,
+	Divider,
+	Grid,
+	makeStyles,
+	Paper,
+	Theme,
+	Typography,
+} from "@material-ui/core";
 import clsx from "clsx";
 import React, { useEffect } from "react";
 import { useInView } from "react-intersection-observer";
 import { AnimateKeyframes } from "react-simple-animate";
 import { RollingNum } from "./component";
-import cosmosSDK from "@carbon-info/assets/icons/cosmosSDK.svg";
+import Banner from "@carbon-info/components/Banner";
+import { DemexLogo } from "@carbon-info/assets";
 
 export interface DataInfo {
-  value: string,
-  description: string,
-  toCountUp: boolean,
+	value: string;
+	description: string;
+	toCountUp: boolean;
 }
 
 const Data: React.FC = () => {
-  const classes = useStyles();
-  const { ref, inView } = useInView({
-    /* Optional options */
-    threshold: 0.4,
-    triggerOnce: true,
-  });
+	const classes = useStyles();
+	const { ref, inView } = useInView({
+		/* Optional options */
+		threshold: 0.4,
+		triggerOnce: true,
+	});
 
-  const widthXs = isWidth("xs");
+	const widthXs = isWidth("xs");
 
-  const { data } = useContentful({
-    contentType: "carbonWebsiteInfo",
-  });
+	const { data } = useContentful({
+		contentType: "carbonWebsiteInfo",
+	});
 
-  const [tableInfo, setTableInfo] = React.useState<DataInfo[]>([]);
-  useEffect(() => {
-    const results: DataInfo[] = [];
-    if (!data && !inView) return;
-    async function fetchDataItems() {
-      const content = await data as any;
+	const [tableInfo, setTableInfo] = React.useState<DataInfo[]>([]);
+	useEffect(() => {
+		const results: DataInfo[] = [];
+		if (!data && !inView) return;
+		async function fetchDataItems() {
```

Comment:
By the time this `useEffect` is executed, the data is already available in JSON format (i.e. inside the `useContentful` hook, we have executed all the necessary `await` operators, so the data should already be available when we run `setData(...)`).
<img width="893" alt="Screenshot 2024-01-08 at 10 41 13 AM" src="https://github.com/Switcheo/carbon-website/assets/65583522/e8884f30-76c5-4567-8af3-f9d32886e356">

As such, there's no need to bind the data processing code in an async function. So your code should look something like this.
```typescript
useEffect(() => {
  const results: DataInfo[] = [];
  if (!data && !inView) return;

  const content = data as any;
  if (content && Array.isArray(content.items)) { ... }
  ...
}, [data])
```